docs/fixers.rst

```diff
@@ -21,6 +21,12 @@ then use the ``--no-six`` flag.
 Fixers use the API defined by 2to3. For details of how this works, and how to
 implement your own fixers, see `Extending 2to3 with your own fixers, at
 python3porting.com <http://python3porting.com/fixers.html>`_.
+``python-modernize`` will try to load fixers whose full dotted-path is specified
+as a ``-f`` argument, but will fail if they are not found. Note that, by
+default, the current directory is not on the Python-Path when you run the
+command; the ``-H``/``--fixers-here`` option is provided in order to add it,
+as a shorthand for explicit manipulation of the ``PYTHONPATH`` environment
+variable.
```

Comment:
I'd cut it off before "as a shorthand". If people know about PYTHONPATH, they don't need telling, and if they don't it's a distraction from what we're trying to explain.